app/assets/javascripts/components/student/pages/lessons_page.es6.jsx

```diff
@@ -1,12 +1,35 @@
 class LessonsPage extends React.Component {
 
+  static get propTypes() {
+    return {
+      studentId: React.PropTypes.number.isRequired,
+    }
+  }
+
   constructor() {
     super();
     this.state = {
       filter: "upcoming",
+      lessons: null,
     };
   }
 
+  componentDidMount() {
+    this.fetchLessons();
+  }
+
+  fetchLessons() {
+    const {studentId} = this.props
```

Comment:
i think we should destructure all property extractions (single and multiple) since it is good style to do so when you are accessing multiple properties of an object (https://github.com/airbnb/javascript#destructuring). if we do `this.props.*` for single properties, and then destructuring for multiple properties, then it will be inconsistent